.travis.yml

```diff
@@ -40,6 +40,13 @@ before_script:
 script:
   - cmake --build . -- -j2
   - ctest -E tpm -VV
+  - ./tool/xtt genkey -v root_priv.bin -b root_pub.bin
+  - ./tool/xtt genkey -v server_priv.bin -b server_pub.bin
+  - ./tool/xtt genrootcert
+  - ./tool/xtt genservercert
+  - ./tool/xtt runserver -d ./data/client/daa_gpk.bin -b ./data/client/basename.bin &
+  - ./tool/xtt runclient -d ./data/client/daa_gpk.bin -c ./data/client/daa_cred.bin -k ./data/client/daa_secretkey.bin -n ./data/client/basename.bin &
+  - sleep 1
```

Comment:
We shouldn't include the ampersand `&` after running the client; we should just run it in the foreground. Running the client in the background means we never actually check the return of the client (e.g. if the client fails, it just fails in the background and we don't know).

That would also remove the need for the `sleep` (though, we probably should add a `sleep` after starting the server and before running the client, just to make sure the server has time to start up).